app/src/main/java/io/github/sustainow/presentation/ui/components/MultiFieldQuestionCard.kt

```diff
@@ -0,0 +1,490 @@
+                            fontSize = 8.sp,
+                            textAlign = TextAlign.Center,
+                        ),
+                        decorationBox = { innerTextField ->
+                            Box(
+                                modifier = Modifier
+                                    .border(1.dp, MaterialTheme.colorScheme.outline, MaterialTheme.shapes.small),
+                                contentAlignment = Alignment.Center
+                            ){
+                                innerTextField()
+                            }
+                        },
+                        value = if(!isExpanded) "Qnt" else "${items.alternatives.size}",
+                        onValueChange = {}
+                    )
+
+                    Spacer(modifier = Modifier.width(10.dp))
+
+                    BasicTextField(
+                        modifier = Modifier
+                            .height(28.dp)
+                            .width(58.dp),
+                        textStyle = TextStyle(
+                            fontSize = 8.sp,
+                            textAlign = TextAlign.Center,
+                        ),
+                        decorationBox = { innerTextField ->
+                            Box(
+                                modifier = Modifier
+                                    .border(1.dp, MaterialTheme.colorScheme.outline, MaterialTheme.shapes.small),
+                                contentAlignment = Alignment.Center
+                            ){
+                                innerTextField()
+                            }
+                        },
+                        value = if(!isExpanded) "Tempo de uso" else "${items.alternatives.sumOf{ it.timePeriod.inWholeHours } / items.alternatives.size}",
+                        onValueChange = {}
+                    )
+
+                    Spacer(modifier = Modifier.width(10.dp))
+
+                    BasicTextField(
+                        modifier = Modifier
+                            .height(28.dp)
+                            .width(30.dp),
+                        textStyle = TextStyle(
+                            fontSize = 8.sp,
+                            textAlign = TextAlign.Center,
+                        ),
+                        decorationBox = { innerTextField ->
+                            Box(
+                                modifier = Modifier
+                                    .border(1.dp, MaterialTheme.colorScheme.outline, MaterialTheme.shapes.small),
+                                contentAlignment = Alignment.Center
+                            ){
+                                innerTextField()
+                            }
+                        },
+                        value = if(!isExpanded) "Valor (W)" else "${items.alternatives.sumOf{ it.value.toDouble() } / items.alternatives.size}",
+                        onValueChange = {}
+                    )
```

Comment:
Quando o meu estiver expandido e nenhum campo do cabeçalho for editável, talvez seja melhor usar Chips do jetpack compose para não ter que definir um `onValueChange` vazio. Na forma atual os text fields não mudam o valor, mas o usuário ainda consegue digitar neles
https://developer.android.com/develop/ui/compose/components/chip?hl=pt-br